testsuite/features/support/env.rb

```diff
@@ -25,7 +25,8 @@
 STDOUT.sync = true
 STARTTIME = Time.new.to_i
 Capybara.default_max_wait_time = ENV['CAPYBARA_TIMEOUT'] ? ENV['CAPYBARA_TIMEOUT'].to_i : 10
-DEFAULT_TIMEOUT = ENV['DEFAULT_TIMEOUT'] ? ENV['DEFAULT_TIMEOUT'].to_i : 250
+# DEFAULT_TIMEOUT temporarily increased from 250 to 350 due to several timeout issues
```

Comment:
As discussed... let's give a try, temporary.
Basically because the step failing with timeout was:
```
When(/^I wait until event "([^"]*)" is completed$/) do |event|
  step %(I wait at most #{DEFAULT_TIMEOUT} seconds until event "#{event}" is completed)
end
```
Which is too generic, and I would discard start refactors to use other steps at that moment of development cycle.